.gitignore

```diff
@@ -13,4 +13,123 @@ UploadStatusFile*
 Tools/kvp_*
 Tools/jq
 Tools/gawk
-DownloadedFiles*
\ No newline at end of file
+DownloadedFiles*.
+./TestExecutionError.log
+./Sysbench.sh
+./summary.log
+./state.txt
+./TestExecution.log
```

Comment:
Inca ai in gitignore fisierele cu ./ (L17-L21)

Reply:
Done